src/System.Net.Security/src/System/Net/Security/SecureChannel.cs

```diff
@@ -997,28 +867,16 @@ internal void ProcessHandshakeSuccess()
                     _trailerSize = streamSizes.Trailer;
                     _maxDataSize = checked(streamSizes.MaximumMessage - (_headerSize + _trailerSize));
                 }
-                catch (Exception e)
+                catch (Exception e) when (!ExceptionCheck.IsFatal(e))
```

Comment:
You're right, the try/catch was already there.

What I should say is, we should consider removing the try/catch since it's only needed for tracing.  That's beyond the scope of this change though.